### ISSUE TITLE
record/replay to/from file

### ISSUE THREAD

**Issue by krzysztofantczak** [sidorares/node-rfb2#14]
Hi,

How can i use that functionality?

**Comment by sidorares** [comment]
Hi! Which format are you going to use? I had an example of "write each rfb packet prefixed with timestamp to file" somewhere but can't find it ( easy to recreate anyway )

also have a look at https://github.com/sidorares/rfbrecord

**Comment by krzysztofantczak** [comment]
Yeah, i was trying to use rfbrecord with https://github.com/pigshell/nhnick - which is a fork of phantomjs improved with vnc server. It didn't worked :( I'm not that familiar with RFB to tell why.. "rect" event was never received there.

**Comment by sidorares** [comment]
are you able to connect with any other vnc viewer?

Also, for screen recording of browser session I'd probably try using built-in screencast via remote debugger - https://chromedevtools.github.io/debugger-protocol-viewer/Page/#method-startScreencast
